test/parallel/test-fs-read-position-validation.mjs

```diff
@@ -91,11 +45,15 @@ async function testInvalid(code, position, internalCatch = false) {
           () => fs.read(fd, { buffer, offset, length, position }, common.mustNotCall()),
           { code }
         );
+        assert.throws(
+          () => fs.read(fd, buffer, { offset, length, position }, common.mustNotCall()),
+          { code }
+        );
       } finally {
-        fs.close(fd, common.mustSucceed());
+        fs.close(fd, common.mustSucceed(resolve));
```

Comment:
This would resolve the promise even if there was an assertion error, right?

Reply:
From my understanding, it will try to resolve an already rejected promise. Which is futile and looks wrong indeed.
Added explicit catch&reject as well just in case.